processes/flood_sar_udf.json

```
@@ -0,0 +1,200 @@
+{
+  "namespace": "user",
+  "description": "Flood extent can be determined using a change detection approach on Sentinel-1 data. In this notebook we have tried in adopting (UN SPIDER's) recommendation practice for computing flood extent by implementing openeo process.Function) UDF.",
```

```suggestion
  "description": "Flood extent can be determined using a change detection approach on Sentinel-1 data. In this process, we have tried adopting UN SPIDER's recommended practice for computing flood extents by implementing an openEO UDF.",
```

Is that what you meant?